docs/user_guide.rst

```diff
@@ -8,22 +8,39 @@ code snippets and check the output yourself.
 
 .. note::
 
-   At the moment IQM does not provide a quantum computing service open to the general public.
-   Please contact our `sales team <https://www.meetiqm.com/contact-us/>`_ to set up your access to an IQM quantum
-   computer.
```

Comment:
So no sales team/contact link is needed anymore?

Reply:
Yes, as there is Resonance, one can learn about Resonance on the main website